src/platform/Kconfig

```diff
@@ -385,6 +385,13 @@ config CAVS_USE_LPRO_IN_WAITI
 	  After waiti exit clock source will be restored.
 	  Choose n if unclear.
 
+config KCPS_DYNAMIC_CLOCK_CONTROL
+	bool "Use KCPS budget to determine DSP clock"
```

Comment:
I think we should explain KCPS at least once in Kconfig, either in summary or in the help text. I'd go with "Use compute budget expressed in Kilo Cycles Per Second (KCPS) to determine DSP clock".

Reply:
Sure, added in "help"